Dockerfile

```diff
@@ -0,0 +1,9 @@
+## relay-dapp dockerfile (This file is called Dockerfile) on server side. ##
+FROM node:12-alpine AS compiler
+RUN apk add --no-cache build-base git bash
+WORKDIR /usr/src/app
+COPY package*.json ./
+COPY .env ./
+COPY . ./
```

Comment:
sorry but with the copy command we run on line #7, do we still need lines #5 and #6?

Reply:
Sorry thats correct, changed.